src/i18n/_locales/en_US/messages.json

```diff
@@ -246,6 +246,23 @@
   "i18n_font_size" : {
     "message" : "FONT SIZE"
   },
+  "i18n_font_selection" : {
+    "message" : "FONT FACE"
+  },
+  "i18n_fonttype-default" : {
+    "message" : "Default"
+  },
+    "i18n_fonttype-odf" : {
+    "message" : "Open Dyslexic Font"
+  },
+    "i18n_font_test-label" : {
+    "message" : "Font Testing Ground"
+  },
+    "i18n_font_test" : {
+    "message" : "This is a test of the font you picked. If you like it, then it is the font for you to read with."
```

Comment:
TODO: decrease the verbosity of this l18n string (too hard to translate), re-word to make clearer.